.github/workflows/continuous-integration.yml

```diff
@@ -121,15 +165,38 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
-    - name: Install opensim-core
-      run: |
-        wget https://prdownloads.sourceforge.net/myosin/opensim-core/opensim-core-latest_linux_Release.zip
-        # -q: quiet
-        unzip -q opensim-core-latest_linux_Release.zip -d ~
 
     - name: Checkout opensim-gui
       uses: actions/checkout@v1
```

Comment:
There are inconsistencies between platforms related to the step for checking out `opensim-gui`.

Reply:
Now all use checkout@v2